docs/reference/pc_repository.md

```diff
@@ -0,0 +1,756 @@
+
+- `previous_names`
+
+To help users find a package, they can be tagged with [labels][]:
+
+- `labels`
+
+[rename]: pc_renaming.html
+[labels]: pc_submitting.html#what-labels-to-use
+
+
+### Quick examples
+
+This is all you need, with all the information pulled from the GitHub, Bitbucket or GitLab repository.
+
+```json
+{
+	"packages": [		
+		{
+			"details": "https://github.com/wbond/sublime_alignment",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": true
+				}
+			]
+		}
+	]
+}
+```
+
+This is more complete, and adds
+a specific name, labels and homepage and donation links.
+
+```json
+{
+	"packages": [
+		{
+			"name": "Solarized Color Scheme",
+			"details": "https://github.com/braver/Solarized",
+			"homepage": "https://ethanschoonover.com/solarized/",
+			"donate": "https://paypal.me/koenlageveen",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"tags": true
+				}
+			]
+		}
+	]
+}
+```
+
+You can find tons of examples in the default repository
+at https://github.com/wbond/package_control_channel/tree/master/repository.
+
+
+### Package metadata without `details`
+
+Package metadata are resolved using `details`,
+where each field can be overridden by explicitly set values.
+If `details` is absent, each release must specify a `base` url.
+In this and other aspects,
+release based packages behave the same as tag based packages.
+
+```json
+{
+	"packages": [
+		{
+			"name": "Package With Asset",
+			"description": "A library with custom download assets",
+			"author": "Jon Doe",
+			"releases": [
+				{
+					"base": "https://github.com/SublimeText/PackageWithAsset",
+					"asset": "Package.With.Asset-${version}-st${st_build}-${platform}.sublime-package",
+					"platforms": ["windows-x64", "linux-x64", "osx-x64"],
+					"sublime_text": ">=4107",
+					"tags": "st4107-"
+				}
+			]
+		},
+		{
+			"name": "My Package",
+			"description": "A locally hosted Sublime Text package",
+			"author": "John Doe",
+			"issues": "https://company.com/software/issues",
+			"releases": [
+				{
+					"platforms": ["windows-x64"],
+					"version": "1.0.0",
+					"url": "file:///c:/absolute/path/to/My Package.sublime-package"
+				},
+				{
+					"platforms": ["linux-x64"],
+					"version": "1.0.0",
+					"url": "./path/relative/to/repository_json/My Package.sublime-package"
+				}
+			]
+		}
+	]
+}
+```
+
+
+## `releases`
+
+All packages must have one or more `releases`.
+Releases reference tags named in accordance with [semver](https://semver.org).
+The value for `tags` can be simply `true` or a string prefix.
+The latter is specifically useful when targeting older versions of ST
+with a specific range of tags.
+
+Note that previously releases based on branches instead of tags was supported,
+but this has been deprecated.
+
+
+### `sublime_text`
+
+A release MUST contain a "sublime_text" version selector.
+Use `*` for all versions,
+or "<3000", "3500-4000", etc.
```

Comment:
```suggestion
or "<3000", "3500 - 4000", etc.
```

I don't know if the spaces are required, but we use them everywhere else there's a range.